CONTRIBUTING.md

```diff
@@ -62,4 +62,4 @@ Travis CI uses the Makefile with the default target, it is recommended to
 run it before submitting your PR. It runs `gofmt -s` (simplify) and `golint`.
 
 The dependencies are managed with `go mod` if you work with the sources under your
-`$GOPATH` you need to set the environment variable `GO111MODULE=on`.
\ No newline at end of file
+`$GOPATH` you need to set the environment variable `GO111MODULE=on`.
```

Comment:
How come this code shows a change yet nothing visible? Did your editor add a newline or something?

Reply:
Probably some diff when merging...